nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/content/HelmContentFacet.java

```diff
@@ -30,6 +33,8 @@
 public interface HelmContentFacet
     extends ContentFacet
 {
+  Iterable<FluentAsset> browseComponentAssets(@Nullable final AssetKind assetKind);
```

Comment:
I don't see any components involved here...

Reply:
Maybe something like `browseAssetByAssetKind()`?

Reply:
oh, I see you just copied it from the orient version. Okay, naming is up to you 😄